bubblewrap.c

```diff
@@ -358,6 +361,11 @@ usage (int ecode, FILE *out)
            "    --info-fd FD                 Write information about the running container to FD\n"
            "    --json-status-fd FD          Write container status to FD as multiple JSON documents\n"
            "    --new-session                Create a new terminal session\n"
+#ifdef WARN_IMPLICIT_SESSION_OPT
+           "    --no-new-session             Don't create a new terminal session\n"
+#else
+           "    --no-new-session             Specify explicitly that you don't want a new terminal session or cancel an earlier --new-session\n"
+#endif
```

Comment:
I think I'd prefer to document this as `Don't create a new terminal session [default]`, whether the option is set or not.